fuzz/client.c

```diff
@@ -50,8 +50,6 @@ int FuzzerInitialize(int *argc, char ***argv)
     ERR_get_state();
     CRYPTO_free_ex_index(0, -1);
     idx = SSL_get_ex_data_X509_STORE_CTX_idx();
-    RAND_add("", 1, ENTROPY_NEEDED);
-    RAND_status();
```

Comment:
No, we cannot make it an empty function and sometimes people do explicitly want to add in their own source of randomness (although they shouldn't).  Doing a dummy add misleads naive people reading the code and is not necessary.